source/recipe-individual/scripts/individual.js

```diff
@@ -0,0 +1,28 @@
+window.addEventListener("DOMContentLoaded", init);
+async function init() {
+}
+
+
+async function fetchIndividualRecipe(id) {
+  let searchString = `https://api.spoonacular.com/recipes/${id}/information?apiKey=03722052291e4f84bce1021acd82624f&includeNutrition=true`;
+  console.log(searchString);  
+  return fetch(searchString)
+    .then((response) => response.json());
+}
+
+window.onload = async function(){
+    console.log('here');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445170)
```suggestion
    console.log("here");
```